compiler/rustc_lint/src/builtin.rs

```diff
@@ -2288,12 +2288,21 @@ impl EarlyLintPass for IncompleteFeatures {
                             n, n,
                         ));
                     }
+                    if HAS_MIN_FEATURES.contains(&name) {
+                        builder.help(&format!(
+                            "this feature has the minimal subset feature `min_{}`, \
```

Comment:
I'm not sure on this exact message - maybe something like this?

> alternatively, consider using `min_{}`, a subset of `{}` which has greater stability and completeness

Reply:
Sounds good! In order to hear other opinions, I'd like to wait for other wg-diagnostics folks for a while.

Reply:
> consider using `min_{}` instead, which is more stable and complete.

Reply:
Wow it's shorter, is it the winner of "diagnostics-golf"? :)

Reply:
If you're alright with that, let's use it.